new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,243 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { getColorClass, productLinks } from '../utils/memberUtils';
+import Button from './Button';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}'s profile picture`}
+      className="rounded-lg w-auto h-auto"
+      width={232}
+      height={232}
+    />
+    <div className="text-left">
+      {enlarged ? (
+        <>
+          <h3>{`${user.firstName} ${user.lastName}`}</h3>
+          <h5 className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</h5>
+        </>
+      ) : (
+        <>
+          <h6>{`${user.firstName} ${user.lastName}`}</h6>
+          <p className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</p>
+        </>
+      )}
+    </div>
+  </>
+);
+
+type MemberCardProps = {
+  user: IdolMember;
+  image: string;
+  selected: boolean;
+  onClick?: () => void;
+};
+
+export const MemberCard = ({ user, image, selected, onClick }: MemberCardProps) => {
+  const baseStyles =
+    'relative p-8 border border-border-1 flex flex-col gap-4 hover:bg-background-2 transition-[background-color] duration-[120ms] has-[:focus]:outline-2 has-[:focus]:outline-offset-3 has-[:focus]:z-10';
```

Comment:
```suggestion
    'relative p-8 border border-border-1 flex flex-col gap-4 hover:bg-background-2 transition-[background-color] duration-[120ms] has-[:focus-visible]:outline-2 has-[:focus-visible]:outline-offset-3 has-[:focus-visible]:z-10';
```


So that the white outline appears the element is focused via `tab` only, not on click.

[See `:focus` vs `:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#focus_vs_focus-visible)